docs/install/containers/recipes/index.md

```diff
@@ -149,6 +152,13 @@ In RelStorage mode, pass the connection DSN.
 docker run -e RELSTORAGE_DSN="dbname='plone' user='plone' host='db' password='password' port='5432'" pack
```

Comment:
```suggestion
docker run -e RELSTORAGE_DSN="dbname='plone' user='plone' host='db' password='password' port='5432'" backend pack
```

The above command assumes that the service that runs the Plone instance is named `backend`.
Otherwise replace `backend` with your container's name.